dotbot/dotbot_simulator.py

```diff
@@ -153,47 +153,45 @@ def update(self):
 
     def advertise(self):
```

Comment:
Why does the bot sends the advertisement? (I understand it may be unrelated to this PR)

Reply:
It is how it notifies its presence to the controller. So it periodically sends advertisement packets so the controller know it's alive. The first one is used by the controller to register a new incoming robot.